### PR TITLE
Update Insights admin role to 'Advisor'

### PR DESCRIPTION
**Pull request by wcmitchell**
Updated configs for Advisor admin role to use correct application name.

Signed-off-by: Chris Mitchell <cmitchel@redhat.com>

### CODE REVIEW

**Comment by wcmitchell** [comment]
This will need some checking to ensure it doesn't break Advisor current setup and needs to wait for the UI to list roles by display_name, which is still in progress. Adding `no merge` flag until those are in place.

File: configs/roles/insights.json [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
+      "display_name": "Advisor administrator",
+      "description": "Perform any available operation against any advisor resource.",
       "system": true,
       "platform_default": true,
-      "version": 6,
+      "version": 7,
       "access": [
         {
-          "permission": "insights:*:*"
+          "permission": "advisor:*:*"
```
Comment (astrozzc) [inline configs/roles/insights.json:12]: Should we modify the one in the permissions folder? We might have to think of a way to delete the old permission "insights:*:*"

Comment (wcmitchell) [reply configs/roles/insights.json:12]: Yep, you're right. I just realized that looking at another PR. For the short term, I'm thinking of just adding an `advisor.json` file in the permissions directory.

**Review by astrozzc — APPROVED** [review]
LGTM!

**Comment by wcmitchell** [comment]
Blockers cleared, sending to CI

**Pull request merged**